src/components/Icon/Icon.module.scss

```diff
@@ -0,0 +1,49 @@
+@import "../../assets/variables.scss";
+
+@mixin icons {
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+}
+
+.icon {
+  @include icons;
+}
+
+.button-icon {
+  @include icons;
+
+  cursor: pointer;
+}
+
+.blue {
+  color: $primary001;
+}
+
+.red {
+  color: $danger001;
+}
+
+.green {
+  color: $success001;
+}
+
+.white {
+  color: $white;
+}
+
+.black {
+  color: $black;
+}
+
+.small {
+  font-size: 1em;
```

Comment:
I'd be a little more careful with `em`s, because it could be troublesome if we pun an icon in some heading or some other element that would affect the `font-size`. I think sticking with `px` or `rem`s could be a little safer. Wdyt?

Reply:
Yes, you are totally right! My mistake with these errors 🙂 Now I fixed everything. Do you see any more code to improve?